module1/homework/grayscaleImages/main.cpp

```diff
@@ -42,9 +42,10 @@ std::array<std::array<uint8_t, 32>, 32> generateNinja() {
 
 int main() {
     auto ninja = generateNinja();
-    // printMap(ninja);
-    auto compressed = compressGrayscale(ninja);
-    auto decompressed = decompressGrayscale(compressed);
+    printMap(ninja);
+    //auto compressed = compressGrayscale(ninja);
```

Comment:
Why comment this code? It will be nice to see that ninja is the same ninja after and before compression, isn't it?